vcu118/bsp.h

```diff
@@ -1,7 +1,7 @@
 #ifndef RISCV_P1_BSP_H
 #define RISCV_P1_BSP_H
 
-#define CPU_CLOCK_HZ 50000000
+#define CPU_CLOCK_HZ 100000000
```

Comment:
Shouldn't we `#ifdef` this to preserve compatibility with P1?

Reply:
Should the software know about P1/2/3?  I figured it should just know the parameters of the system it's being run on (which is why I was talking earlier about having some kind of board config file).  I changed it to use `#ifndef CPU_CLOCK_HZ` so that it can be overridden for systems other than P2, but I can change it to use some kind of phase variable instead of that would be better.

Reply:
Ideally, the runtime would have an SOC/BOARD variable that would set `-D CPU_CLOCK_MHZ=XXX` appropriately at build time for the P1/P2/P3 processors. The GFE does this for FreeRTOS.